### PR TITLE
Support Qt5 with fallback to Qt4

### PR DESCRIPTION
**Pull request by 4gra**

### CODE REVIEW

**Comment by 4gra** [comment]
Untested on Qt4, but I don't see why it wouldn't work!  Fails as you'd expect if neither is available.

File: README.md [hunk 1]
```diff
@@ -72,7 +75,10 @@ Example using save/load button and keeping the dialog open when pressing ok
 -----------------------------------------------------------------------------------------------------
     import argparse
     import sys
-    from PyQt4 import QtGui
+        try:
```
Comment (shimpe) [inline README.md:78]: git diff seems to show an indentation error here ?

Comment (4gra) [reply README.md:78]: ha, quite right; it is a readme in my defence so I did it freehand!

Comment (4gra) [reply README.md]: fixed.

**Pull request merged**